### PR TITLE
Allow KnpLabs/gaufrette 0.8

### LINKED ISSUE #1509: Handling PHP 7.3 support and KnpLabs/Gaufrette
Though not explicitely stated through composer requirements, KnpLabs/Gaufrette 0.7 doesn't support PHP 7.3. Version 0.8 supports PHP 7.3 but drops support for every version before PHP 7.1.

So how is supposed to be handled by the Media bundle?

Would it be okay to just allow KnpLabs/Gaufrette ^0.8 or would this mean that support for older PHP version had to be dropped as well?

### PR DESCRIPTION
**Pull request by IwfY**
Closes #1509 

## Changelog

```markdown
### Changed
- Allow KnpLabs/Gaufrette 0.8
```

### CODE REVIEW

**Comment by greg0ire** [comment]
Please associate your email address with your Github account, or change the email in your commits to an address already associated with it.

File: composer.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
         "guzzlehttp/psr7": "^1.0",
         "imagine/imagine": "^0.6 || ^0.7",
         "jms/serializer-bundle": "^1.0 || ^2.0",
-        "knplabs/gaufrette": "^0.3 || ^0.4 || ^0.5 || ^0.6 || ^0.7",
+        "knplabs/gaufrette": "^0.3 || ^0.4 || ^0.5 || ^0.6 || ^0.7 || ^0.8",
```
Comment (greg0ire) [inline composer.json]: Sigh… when are they going to release version 1? Before the 120 chars limit I hope!

**Comment by greg0ire** [comment]
Blocked by #1511

**Comment by greg0ire** [comment]
Do we usually label this minor or patch?

File: composer.json (hunk 1 above)
Comment (greg0ire) [reply composer.json]: Lol I already complained last time: #1477

**Comment by OskarStark** [comment]
Would say minor

**Comment by greg0ire** [comment]
> Please associate your email address with your Github account, or change the email in your commits to an address already associated with it.

Looks like this is fixed :)

**Pull request merged**